### PR TITLE
add id to draggable-section

### PR DESCRIPTION
**Pull request by omrihecht**
to be used for instance to scroll to an element

### CODE REVIEW

File: projects/ui-framework/src/lib/layout/sortable-collapsible-sections/sortable-collapsible-sections.component.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     cdkDropList>
 
   <li *ngFor="let section of UISections; trackBy: trackId; let index=index"
+      id="{{section.id | json}}"
```
Comment (omrihecht) [inline projects/ui-framework/src/lib/layout/sortable-collapsible-sections/sortable-collapsible-sections.component.html:7]: `| json` makes string, not sure what happens when you have `id="'0'"`

Comment (galloween) [reply projects/ui-framework/src/lib/layout/sortable-collapsible-sections/sortable-collapsible-sections.component.html:7]: Not sure why you would put json pipe on a string or number. 
What’s wrong with id=“0” ? 
If you worry about them not being unique enough you can prefix it with something. id=“‘bcss-’+section.id”

Comment (omrihecht) [reply projects/ui-framework/src/lib/layout/sortable-collapsible-sections/sortable-collapsible-sections.component.html:7]: i thought you could not set id=number

Comment (omrihecht) [reply projects/ui-framework/src/lib/layout/sortable-collapsible-sections/sortable-collapsible-sections.component.html]: seems to be ok, though, removing | json

**Pull request merged**